### PR TITLE
[TEST]: Path check: Diverse: Check all flows from group

### PR DESCRIPTION
**Pull request by yuliiamir**

### CODE REVIEW

File: src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/SwitchPairs.groovy [hunk 1]
```diff
@@ -42,4 +51,9 @@ class SwitchPairs {
         assumeFalse(switchPairs.isEmpty(), "No suiting switch pair found")
         return switchPairs.first()
     }
+
+    SwitchPair pairWithDesiredSwitch(Switch sw) {
+        assumeFalse(switchPairs.isEmpty(), "No suiting switch pair found")
```
Comment (pkazlenka) [inline src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/SwitchPairs.groovy:56]: You don't need these assumptions.
Refactor method to have signature like  `SwitchPairs includeSwitch(Switch sw)`  and use 'random()' method from this class in test.

File: src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/SwitchPairs.groovy [hunk 2]
```diff
@@ -29,6 +31,13 @@ class SwitchPairs {
     SwitchPairs nonNeighbouring() {
         return new SwitchPairs(switchPairs.findAll { it.paths.min { it.size() }?.size() > 2})
     }
+    SwitchPairs neighbouring() {
+        return new SwitchPairs(switchPairs.findAll { it.paths.min { it.size() }?.size() == 2})
+    }
+
+    SwitchPairs excludePair(SwitchPair excludePair) {
```
Comment (pkazlenka) [inline src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/SwitchPairs.groovy:38]: Would be better to have an ability to pass list of switch pairs here.

**Review by pkazlenka — CHANGES_REQUESTED** [review]

File: src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/SwitchPairs.groovy (hunk 1 above)
Comment (yuliiamir) [reply src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/SwitchPairs.groovy]: Done.

File: src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/SwitchPairs.groovy (hunk 2 above)
Comment (yuliiamir) [reply src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/SwitchPairs.groovy]: Done.

**Review by pkazlenka — APPROVED** [review]

**Review by dmitrii-beliakov — APPROVED** [review]

**Pull request merged**